kernel/src/main/java/org/kframework/backend/kore/ModuleToKORE.java

```diff
@@ -1535,15 +1535,35 @@ private Att addKoreAttributes(Production prod, SetMultimap<KLabel, Rule> functio
         String format = att.getOptional("format").orElse(Formatter.defaultFormat(prod.items().size()));
         int nt = 1;
         boolean hasFormat = true;
+        boolean printName = true;
+        boolean printEllipses = false;
+        for (int i = 0; i < prod.items().size(); i++) {
+          if (prod.items().apply(i) instanceof NonTerminal && ((NonTerminal) prod.items().apply(i)).name().isEmpty()) {
+            printName = false;
+            break;
+          }
+        }
```

Comment:
```suggestion
        boolean printName = stream(prod.items()).noneMatch(pi -> pi instanceof NonTerminal && ((NonTerminal) pi).name().isEmpty());
        boolean printEllipses = false;
```
This can be simplified greatly if you use streams.